Team_Roster/app/templates/teamContainer.html

```diff
@@ -7,5 +7,6 @@ <h1>{{ title }}</h1>
 <button onclick="window.location.href='/{{teamId}}/depthChart?year={{year}}'">Depth Chart</button>
 
 {% block teamData %} {% endblock %}
+{% block pitchingData %}{% endblock %}
```

Comment:
why is there a separate block for pitching data? Everything should be able to go within the teamData block.